spec/spec_helper.rb

```diff
@@ -11,9 +11,11 @@
 require_relative '../helpers/DocumentServerHelper'
 require_relative '../helpers/file_helper'
 require_relative '../helpers/pretests_check'
+require_relative '../helpers/image_helper'
```

Comment:
This code and code about USE_JWT is not related to those PR, better to create new ones for those